grml-debootstrap

```diff
@@ -879,10 +863,10 @@ prompt_for_mirror()
                0 0 "$MIRROR")" || bailout
 
   else # CHOOSE_MIRROR == local
-     [ -n "$ISO" ] || ISO='/mnt/mirror'
-     ISO="$(dialog --stdout --title "${PN}" --inputbox \
-               "Please enter directory name you would like to use for installing packages." \
-               0 0 "$ISO")" || bailout
+     MIRROR='file:///mnt/mirror'
+     MIRROR="$(dialog --stdout --title "${PN}" --inputbox \
+               "Please enter the directory in file:/// URL form you would like to use for installing packages." \
+               0 0 "$MIRROR")" || bailout
```

Comment:
this is the only unclear issue for me that we might wanna verify, other than this LGTM (probably not worth a debian/NEWS entry?)

Reply:
removed this part